documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -3107,7 +3110,7 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Category** : SDK Error
 
-**Applies to** : ARM OpenAPI(swagger) specs
+**Applies to** : ARM and Data Plane OpenAPI(swagger) specs
```

Comment:
wonder if we could change all "ARM" to "control plane" or "management plane"

Reply:
I think it's aligned with AutoRest  configuration 'openapi-type'

Reply:
Why we should be aligned with AutoRest configuration? In most of our wikis, and OpenAPI Hub, we use "control plane" or "management plane" versus "data-plane". Why we are not aligned with these?

Reply:
Linter is an AutoRest plugin and in readme.md , we must specify the openapi type with 'arm' and  'data-plane'. Also in the linter code , it used 'arm' not 'management plane' or 'control plane'  .

Reply:
See.